go/vt/wrangler/tablet.go

```diff
@@ -46,7 +46,7 @@ func (wr *Wrangler) DeleteTablet(ctx context.Context, tabletAlias *topodatapb.Ta
 	}
 
 	wasPrimary, err := wr.isPrimaryTablet(ctx, ti)
-	if err != nil {
+	if err != nil && !topo.IsErrType(err, topo.NoNode) {
```

Comment:
If we find that the shard is no longer present, should we just exit the function at this point and not proceed or do we want it to try to cleanup the topo tablet record?  The `topotools.DeleteTablet()`, which will get called if we ignore the error, updates some shard related topo entries and it is not clear if it will fail in case the shard has already been deleted.